src/common/header/components/TermSelect.tsx

```diff
@@ -42,10 +42,23 @@ export function TermSelect(): JSX.Element {
     }
   };
 
+  let colorMap: { [key: string]: string } = {};
+  colorMap['09'] = mode('green.500', 'green.300');
+  colorMap['01'] = mode('blue.500', 'blue.300');
+  colorMap['05'] = mode('yellow.500', 'yellow.300');
```

Comment:
I'd flip these such that the darker colors are used when on dark mode as it contrasts better with the text